### DISCUSSION TITLE
Create Island name

### DISCUSSION THREAD

**Discussion by TimDegraaf** [BG-Software-LLC/SuperiorSkyblock2#803] [Denied Feature Requests] [answered]
When creating a island name, make so the players can create the name by running something like /is create then the name in chat. And it tells them to create the name. Maybe add npc support for creating a island name as well. This would help new players create islands.

**Comment by OmerBenGera** [comment, marked as answer]
`/is create <island-name>` is a thing.
If people can't understand how to read commands usage, then they should not play the game I guess...
All the commands of the plugin have arguments.

Denied, thanks for submitting tho!
